SurveyAppTests/Tests/Modules/SurveyDetail/SurveyDetailRouterSpec.swift

```diff
@@ -27,6 +28,22 @@ final class SurveyDetailRouterSpec: QuickSpec {
                 router.view = viewController
                 _ = viewController.view
             }
+            
+            describe("its showSurveyQuestion is called") {
+                context("when the parameter is valid id and question") {
+                    beforeEach {
+                        let id = "d5de6a8f8f5f1cfe51bc"
+                        let questions: [SurveyQuestion] = JSON.SurveyService.surveyQuestionModelList.decoded()
```

Comment:
instead of using `decodable` I think it's better to create Dummy Object. You can remove `Decodable` from the classes since you won't need them.
```

@testable import DeeMoneyNeo

extension SurveyQuestion {

    static var dummy: SurveyQuestion {
        SurveyQuestion(
           ...
        )
    }
}
```

```
let questions: [SurveyQuestion] = [.dummy]
```